### PR TITLE
LTD-4645-healthcheck-app-readd

### PR DESCRIPTION
**Pull request by seijihg**
a bit of cleaning up since few things were removed when merged.

### CODE REVIEW

File: healthcheck/apps.py [hunk 1]
```diff
@@ -10,11 +10,9 @@ def ready(self):
         from .checks import (
             MailboxAuthenticationHealthCheck,
             LicencePayloadsHealthCheck,
-            ManageInboxTaskHealthCheck,
             PendingMailHealthCheck,
         )
 
         plugin_dir.register(MailboxAuthenticationHealthCheck)
         plugin_dir.register(LicencePayloadsHealthCheck)
-        plugin_dir.register(ManageInboxTaskHealthCheck)
```
Comment (currycoder) [inline healthcheck/apps.py:19]: Is the code for `ManageInboxTaskHealthCheck` still present?  Would suggest removing it if so

Comment (seijihg) [reply healthcheck/apps.py:19]: ah no, it was removed when @Tllew merged his PR

Comment (Tllew) [reply healthcheck/apps.py:19]: I'm not sure how you can check the manageinbox task as it's all converted to celery, but if it works then happy to keep it in

**Review by Tllew — APPROVED** [review]

File: healthcheck/apps.py (hunk 1 above)
Comment (currycoder) [reply healthcheck/apps.py:19]: We decided to drop the task, but a little confused that it's still referenced here

Comment (saruniitr) [reply healthcheck/apps.py:19]: Yeah it is a bit confusing to see these references but the code is not there.

**Review by saruniitr — APPROVED** [review]
Suggest deploying to an env and checking before merging this.

**Comment by seijihg** [comment]
> Suggest deploying to an env and checking before merging this.

done checked on UAT and DEMO both working fine

**Pull request merged**